source/standards/vulnerability-disclosure.html.md.erb

```diff
@@ -31,7 +31,15 @@ You should use <https://vdp.cabinetoffice.gov.uk/.well-known/security.txt>
 in either:
 
 - the origin for your site's `/.well-known/security.txt`
-- the destination of a 302 redirect for `/.well-known/security.txt`
+- the destination of a redirect for `/.well-known/security.txt`
+
+A note on the redirect mechanism, try implementing in the following order to
+ensure the best capability with all user agents:
```

Comment:
I personally wouldn't line-break these, but not sure what the GDS Way is for this :upside_down_face: 

```suggestion
A note on the redirect mechanism, try implementing in the following order to ensure the best capability with all user agents:
```